lexicons/place/stream/default/metadata.json

```
@@ -0,0 +1,72 @@
+{
+  "lexicon": 1,
+  "id": "place.stream.default.metadata",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Default metadata record for livestream including content warnings, rights, and distribution policy",
+      "key": "self",
+      "record": {
+        "type": "object",
+        "properties": {
+          "contentWarnings": {
```

I think we agreed on singular?

```suggestion
          "contentWarning": {
```

But then `renditions` array here is plural https://github.com/hyphacoop/streamplace/blob/next/lexicons/place/stream/defs.json